### PR TITLE
Fixed tab request approval issue

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/tab-request-approval/csharp/TabRequestApproval/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -202,7 +202,7 @@
                  title: $('#title').val(),
                  description: $('#description').val(),
                  userName: username,
-                managerName: person[0].userPrincipalName,
+                managerName: person[0].scoredEmailAddresses[0].address,
```
Comment (ChetanSharma-msft) [inline samples/tab-request-approval/csharp/TabRequestApproval/Views/Home/Index.cshtml:205]: Kindly rename variable as userName or personaName

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comment

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Comment by Prithvi-MSFT** [comment]
Build status  - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5411190102

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**